tasks/main.yml

```diff
@@ -69,8 +73,9 @@
   register: gpg_list_keys
   when: caddy_pgp_verify_signatures
   tags: skip_ansible_lint
-  failed_when: "gpg_list_keys is failed and ('gpg: error reading key: No public key' not in gpg_list_keys.stderr and
-                                             'gpg: error reading key: public key not found' not in gpg_list_keys.stderr)"
+  failed_when: "gpg_list_keys is failed and
```

Comment:
Would this work?:
```
failed_when: 
  - gpg_list_keys is failed
  - 'gpg: error reading key: No public key' not in gpg_list_keys.stderr
  - 'gpg: error reading key: public key not found' not in gpg_list_keys.stderr
```

Reply:
Had a quick read & it seems like this will do the same thing with much cleaner syntax so I'll update it to use this & retest. Thanks for the suggestion.